sup3r/pipeline/forward_pass.py

```diff
@@ -410,6 +410,26 @@ def get_hr_slices(slices, enhancement, step=None):
             hr_slices.append(slice(start, stop, step))
         return hr_slices
 
+    @property
+    def chunk_lookup(self):
+        """Get a 3D array with shape
+        (n_spatial_1_chunks, n_spatial_2_chunks, n_temporal_chunks)
+        where each value is the chunk index."""
+        n_s1 = len(self.s1_lr_slices)
+        n_s2 = len(self.s2_lr_slices)
+        n_t = self.n_temporal_chunks
+        lookup = np.arange(self.n_chunks).reshape((n_t, n_s1, n_s2))
+        lookup = np.transpose(lookup, axes=(1, 2, 0))
```

Comment:
How fast is the reshape? Do we want to only create this lookup array once?

Reply:
oh super fast, it's just the low res data shape. I wouldnt worry about it.

Reply:
Ok cool